src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/AdmissionStateCellModel.swift

```diff
@@ -29,7 +29,12 @@ final class AdmissionStateCellModel {
 	}
 
 	var description: String? {
-		healthCertifiedPerson.dccWalletInfo?.admissionState.longText?.localized(cclService: cclService)
+		if healthCertifiedPerson.isAdmissionStateChanged {
+			return "Ihr Status hat sich geändert. Ihre Zertifikate erfüllen jetzt die 2G-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht.\nMehr Informationen zu Ihrem Status im FAQ."
+			// return healthCertifiedPerson.dccWalletInfo?.admissionState.stateChangeNotificationText?.localized(cclService: cclService)
```

Comment:
I think this was just meant for testing purposes :D

Reply:
yes, thanks for noticing, i pushed again